airbyte-workers/build.gradle

```diff
@@ -62,3 +64,31 @@ application {
     mainClass = mainClassName
     applicationDefaultJvmArgs = ['-XX:MaxRAMPercentage=75.0']
 }
+
+task copyDocker(type: Copy) {
```

Comment:
i really like how this structure turned out!

Reply:
it is even better now because it is factorize in a single place (the root gradle file.)